projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -91,6 +92,20 @@ function register_block() {
 		)
 	);
 
+	register_post_meta(
+		'post',
+		META_NAME_FOR_POST_DONT_EMAIL_TO_SUBS,
```

Comment:
Note that this was old meta field I'm re-using for backwards compatibility, but it was never registered like this:

https://github.com/search?q=repo%3AAutomattic%2Fjetpack%20_jetpack_dont_email_post_to_subs&type=code